ha/cli/commands.py

```diff
@@ -119,4 +119,26 @@ def add_args(component_parser):
         status_parser.add_argument("service_name",
             help="Service name", action="store")
         status_parser.add_argument("--node",
-            help="Node name")
\ No newline at end of file
+            help="Node name")
+
+class SupportBundleCommand(Command):
+    @staticmethod
+    def add_args(component_parser):
+        """
+        Add parser for support bundle command
+
+        Args:
+            component_parser (<class 'argparse._SubParsersAction'>): argparse to add subparser.
+        """
+        bundle_parser =  component_parser.add_parser("support_bundle",
+                                help = "Support Bundle Manage")
```

Comment:
Please look at all the help messages and use proper case for each word.
"Support Bundle Manage"  -> Can we add better help text.

Reply:
Corrected all cmd help to proper messages..